test/setup/main.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "project" {
+  source  = "terraform-google-modules/project-factory/google"
+  version = "~> 3.0"
+
+  name              = "ci-{{ cookiecutter.module_name|replace('-', '_') }}"
+  random_project_id = "true"
```

Comment:
It boolean variable, so quotes should be removed

Reply:
@ingwarr, thank you for review! [Fixed](https://github.com/terraform-google-modules/terraform-google-log-export/compare/07b614e0d09443a93a5e1eae9dbc3f053fd832c9..2976460f55f997027cbd58e300dfe5d00e427140)

Reply:
Except you didn't fix it...

Reply:
Good eye, @morgante! [Fixed](https://github.com/terraform-google-modules/terraform-google-log-export/compare/d9bd46a326a17ba2bd80b9ef2fcd09ae049865e7..21c7079f7f4585d8304ce8988f0ec8ecffc2fdf8)
But actually, shouldn't we raise this in [the module](https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/test/setup/main.tf#L22) that came from ?
Thanks!